rust-runtime/aws-smithy-runtime-api/src/client/behavior_version.rs

```diff
@@ -3,17 +3,23 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-//! Behavior Major version of the client
+//! Behavior version of the client
 
-/// Behavior major-version of the client
+/// Behavior version of the client
 ///
 /// Over time, new best-practice behaviors are introduced. However, these behaviors might not be
 /// backwards compatible. For example, a change which introduces new default timeouts or a new
 /// retry-mode for all operations might be the ideal behavior but could break existing applications.
-#[derive(Clone)]
+#[derive(Copy, Clone, PartialEq)]
 pub struct BehaviorVersion {
```

Comment:
We should probably add a section to the developer guide on behavior versions, what they are and guidance on how to choose, and what changed in each.

Reply:
It's there - https://docs.aws.amazon.com/sdk-for-rust/latest/dg/client.html